tests/manager/api/controller/test_patron_activity_history.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import annotations
+
+import pytest
+
+from palace.manager.sqlalchemy.model.patron import Patron
+from tests.fixtures.api_controller import ControllerFixture
+from tests.fixtures.database import DatabaseTransactionFixture
+from tests.fixtures.services import ServicesFixture
+
+
+class PatronActivityHistoryFixture(ControllerFixture):
```

Comment:
`ControllerFixture` does a whole lot of setup that you probably don't need here. In any new fixtures we have been writing, I'm trying not to inherit from that class.

You should be able to use `FlaskAppFixture` with a small fixture that instantiates the controller to test this. A good example is the tests for [test_discovery_services.py](https://github.com/ThePalaceProject/circulation/blob/ebcddcab60487e7f9a763b27296c39aca04f2649/tests/manager/api/admin/controller/test_discovery_services.py).